pkg/identity/restoration/local_identity_restorer.go

```diff
@@ -62,10 +72,31 @@ type LocalIdentityRestorer struct {
 }
 
 func newLocalIdentityRestorer(params localIdentityRestorerParams) *LocalIdentityRestorer {
-	return &LocalIdentityRestorer{
+	restorer := &LocalIdentityRestorer{
 		params:        params,
 		restoredCIDRs: nil, // will be initialized during restoration
 	}
+
+	params.JobGroup.Add(job.OneShot("release-local-identities", func(ctx context.Context, health cell.Health) error {
```

Comment:
nit: the passed `cell.Health` is unused, so mark it as such:

```suggestion
	params.JobGroup.Add(job.OneShot("release-local-identities", func(ctx context.Context, _ cell.Health) error {
```

Reply:
thanks, fixed